.circleci/ssh-config

```diff
@@ -0,0 +1,9 @@
+Host prod-server
+  HostName testing.govstack.global
+  User ubuntu
+  IdentityFile ~/.ssh/id_rsa
+
+Host staging-server
+  HostName staging.testing.govstack.global
+  User ubuntu
+  IdentityFile ~/.ssh/id_rsa_staging
```

Comment:
we should drop identity file from here I believe, we just will use the default key from Circle (same as on line 4, but I don't think you need to call it out)